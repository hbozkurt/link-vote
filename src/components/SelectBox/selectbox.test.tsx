import * as React from "react";
import { create } from "react-test-renderer";
import { SelectBox } from "./index";

describe("Button component", () => {
  test("Matches the snapshot", () => {
    const select = create(
      <SelectBox
        onChange={() => {}}
        options={[{ id: "1", label: "yes" }, { id: "2", label: "no" }]}
      />
    );
    expect(select.toJSON()).toMatchSnapshot();
  });
});
