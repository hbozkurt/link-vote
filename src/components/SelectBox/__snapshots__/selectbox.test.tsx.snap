// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<div
  className="relative inline-block w-64"
>
  <select
    className="appearance-none border w-64 h-8 bg-gray-100"
    onChange={[Function]}
  >
    <option
      value="1"
    >
      yes
    </option>
    <option
      value="2"
    >
      no
    </option>
  </select>
  <span
    className="absolute w-8 h-8 p-2 top-0 right-0 text-lg"
  >
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 320 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"
      />
    </svg>
  </span>
</div>
`;
